bindings/generator/api/common.py

```diff
@@ -17,6 +17,14 @@ class Result(Generic[OK, ERR]):
     pass
 
 
+class Enum:
+    pass
+
+
+class EnumUnit:
```

Comment:
```suggestion
class EnumItemUnit:
```

(to be consistent with `VariantItemUnit`)

Reply:
done